bedrock/mozorg/redirects.py

```diff
@@ -623,4 +623,7 @@ def decider(request, **kwargs):
 
     redirect(r'^(careers|jobs)/?$', 'https://careers.mozilla.org/'),
     redirect(r'^join/?$', 'https://donate.mozilla.org/'),
-)
+
+    #Bug 1262593
+    redirect(r'^unix/remote\.html$', 'http://www-archive.mozilla.org/unix/remote.html'),
+)
```

Comment:
Nit: new line at end of file please (you should be able to configure your code editor to do this automatically)